index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="starting-page">
+        <header class="starting-page>
+            <nav id="navigator_bar">
+                <a class="logo" href="#"><h2>Mylogo</h2></a>
+                <nav class="navigator_menu"><img src="assets/Enabled.png">
+                </nav>
+            </nav>
+        </header>
+        <section id="section-presentation">
+            <h4 class="short-presentation">I'm shri sakthi <br> glad to see you...</h4>
+            <p>I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                on’t hestiate to contact me.
+            </p>
+            <footer>
+                <h3>LET'S CONNECT</h3>
+                <div id="contact">
+                    <ul>
+                        <li class="icon"><a href="https://www.linkedin.com/in/venkatesh-shri-sakthi/" ><img src="assets/Linkedin icon.png" ></a></li>
+                        <li class="icon"><a href="https://github.com/shri-sakthi-v"><img src="assets/Vector (1).png" ></a></li>
+                        <li class="icon"><a href="https://twitter.com/shrisakthi"><img src="assets/Vector.png" ></a></li>
+                        <li class="icon"><a href="https://twitter.com/shrisakthi"><img src="assets/storm.png" alt="#"></a></li>
+                        <li class="icon"><a href="https://twitter.com/shrisakthi"><img src="assets/mail.png" alt=""></a></li>
+                    </ul>
+                </div>
+            </footer>
+
+        </section>
+    </div>
```

Comment:
- [ ] [Optional] Please try to cover the whole height of the page. It is covering only half of the height.